CHANGELOG.md

```diff
@@ -2,14 +2,21 @@
 
 ## Master (Unreleased)
 
+## 3.0.0.pre (2024-06-05)
+
 - Remove extracted cops in `Capybara`, `FactoryBot` and `Rails` departments. ([@ydah])
 - Remove `RuboCop::RSpec::Language::NodePattern`. ([@ydah])
 - Remove `RSpec/FilePath` cop. ([@ydah])
-- Add new `RSpec/ExpectInLet` cop. ([@yasu551])
 - Remove `RSpec/Capybara/FeatureMethods` cop. If you are using this cop, change it to use `RSpec/Dialect`. ([@ydah])
 - Support `AutoCorrect: contextual` option for LSP. ([@ydah])
```

Comment:
Can we release this support first since I want to release it in the v2.x?

Reply:
Done in v2.31.0